packages/sanity/src/media-library/plugin/VideoInput/VideoPreview.tsx

```diff
@@ -41,9 +40,6 @@ export function VideoPreview(props: VideoAssetProps) {
   const asset = value?.asset
   const sourcesFromSchema = schemaType.options?.sources
   const accept = get(schemaType, 'options.accept', '')
-  const isStaging = useClient({apiVersion: DEFAULT_API_VERSION})
```

Comment:
hm, now this will cause downstream failure if you're in the context of a production project, right?

Reply:
No, we can't force apihost to sanity.work because we don't know whether a workspace is a staging or production. So if we force apihost to sanity.work, that will fail for non-staging projectIds

Reply:
the complication here is a scenario where you have a mix of both production and staging projects/datasets